### ISSUE TITLE
Game.GetUserInput Does Not Set Window Title

### ISSUE THREAD

**Issue by BlocTheWorker** [Halen84/ScriptHookRDR2DotNet-V2#5]
## **Problem**
It looks like Game.GetUserInput is not changing the Window title. Tried:
```csharp
Game.GetUserInput("Test title", 128);
```
or 
```csharp
Game.GetUserInput("Test title", "default text", 128);
```
Same result. 

## **Possible Reason**
Looks like string needs to be registered. 
`GetUserInput` is calling ` MISC.DISPLAY_ONSCREEN_KEYBOARD(0, windowTitle, "", defaultText, "", "", "", maxLength + 1);` and it's calling `Function.Call(306580201096983501uL, textType, windowTitle, p2, defaultText, defaultConcat1, defaultConcat2, defaultConcat3, maxInputLength);` - based on [this catalog ](https://docs.fivem.net/natives/?_0x00DC833F2568DBF6) possible titles are 
```
windowTitle's  
-----------------  
CELL_EMAIL_BOD	=	"Enter your Eyefind message"  
CELL_EMAIL_BODE	=	"Message too long. Try again"  
CELL_EMAIL_BODF	=	"Forbidden message. Try again"  
CELL_EMAIL_SOD	=	"Enter your Eyefind subject"  
CELL_EMAIL_SODE	=	"Subject too long. Try again"  
CELL_EMAIL_SODF	=	"Forbidden text. Try again"  
CELL_EMASH_BOD	=	"Enter your Eyefind message"  
CELL_EMASH_BODE	=	"Message too long. Try again"  
CELL_EMASH_BODF	=	"Forbidden message. Try again"  
CELL_EMASH_SOD	=	"Enter your Eyefind subject"  
CELL_EMASH_SODE	=	"Subject too long. Try again"  
CELL_EMASH_SODF	=	"Forbidden Text. Try again"  
FMMC_KEY_TIP10	=	"Enter Synopsis"  
FMMC_KEY_TIP12	=	"Enter Custom Team Name"  
FMMC_KEY_TIP12F	=	"Forbidden Text. Try again"  
FMMC_KEY_TIP12N	=	"Custom Team Name"  
FMMC_KEY_TIP8	=	"Enter Message"  
FMMC_KEY_TIP8F	=	"Forbidden Text. Try again"  
FMMC_KEY_TIP8FS	=	"Invalid Message. Try again"  
FMMC_KEY_TIP8S	=	"Enter Message"  
FMMC_KEY_TIP9	=	"Enter Outfit Name"  
FMMC_KEY_TIP9F	=	"Invalid Outfit Name. Try again"  
FMMC_KEY_TIP9N	=	"Outfit Name"  
PM_NAME_CHALL	=	"Enter Challenge Name" 
```
In other words, it might required to be registered as well.

### **Possibly related to:** #4

**Comment by Halen84** [comment]
Yea, custom strings cannot be used on the onscreen keyboard because the strings must be registered in the string database. This won't work in C++ either. To get custom text on the onscreen keyboard, you must hook the `HUD::_GET_LABEL_TEXT_2` C++ native function to return the string you want. `VAR_STRING` wont work in this case.

Example valid strings: `STABLE_RENAME_MOUNT_PROMPT`, `STABLE_RENAME_MOUNT_DESC`.

GTA V strings aren't the same for RDR3, so I highly doubt they will work. 

For strings that work on the onscreen keyboard, see the RDR3 decompiled scripts and look for `DISPLAY_ONSCREEN_KEYBOARD` function calls.
